Cargo.toml

```diff
@@ -53,6 +53,12 @@ panic = "abort"
 codegen-units = 1
 lto = "fat"
 
+[patch.crates-io]
+op-alloy-genesis = { git = "https://github.com/alloy-rs/op-alloy", branch = "main" }
+op-alloy-protocol = { git = "https://github.com/alloy-rs/op-alloy", branch = "main" }
+op-alloy-consensus = { git = "https://github.com/alloy-rs/op-alloy", branch = "main" }
+op-alloy-rpc-types-engine = { git = "https://github.com/alloy-rs/op-alloy", branch = "main" }
```

Comment:
I _think_ the issue may be that the `op-alloy` release used `0.6.0` alloy deps and didn't include the `0.6.1` releases.

If this passes CI, I will release a new round of patches for `op-alloy` and bump here.